artifacttemplates/http%3A%2F%2Fopentosca.org%2Fartifacttemplates/TomcatApplication_WAR-install-w1/README.md

```diff
@@ -0,0 +1,20 @@
+# TomcatApplication_WAR-connectTo IA [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
```

Comment:
It seems that the two Readmes from TomcatApplication_WAR-install-w1 and TomcatApplication_WAR-connectTo-w1 are exchanged by accident